app/src/main/java/nl/tudelft/bbw/BlockChainAPI.java

```diff
@@ -22,133 +21,103 @@
+     * so you can look up his/her contact & find his/her genesis block
      *
-     * @return List of blocks forming this user contact list.
+     * @param acquaintance object of your pairing partner
+     * @throws BlockAlreadyExistsException when a block already exists
+     * @throws HashException               when the hash methods are not available
      */
-    public static List<Block> getMyContacts() {
-        return blockController.getBlocks(owner);
-    }
-
-    /**
-     * Getter method to get the BlockChainAPI user's name
-     *
-     * @return name of the BlockChainAPI user
-     */
-    public static String getMyName() {
-        return owner.getName();
-    }
-
-    /**
-     * Getter method to get the BlockChainAPI user's Iban
-     *
-     * @return Iban number of the BlockChainAPI user
-     */
-    public static String getMyIban() {
-        return owner.getIban();
-    }
-
-    /**
-     * Getter method to get the BlockChainAPI user's Public Key
-     *
-     * @return Public Key  of the BlockChainAPI user
-     */
-    public static EdDSAPublicKey getMyPublicKey() {
-        return owner.getPublicKey();
-    }
-
-
-    public static void addAcquaintanceMultichain(Acquaintance acquaintance) throws BlockAlreadyExistsException, HashException {
-        //Adding his database into your database (so you can look up his contacts)
+    public static void addAcquaintanceMultichain(Acquaintance acquaintance) throws
+            BlockAlreadyExistsException, HashException {
         blockController.addMultichain(acquaintance.getMultichain());
     }
 
-
     /**
-     * Method to add a contact to your chain
-     *
-     * @throws HashException               When creating a block results in an error
-     * @throws BlockAlreadyExistsException when adding a block results in an error
+     * @param user which you want to add to your contact list
+     * @return The block which represent the contact in your chain
+     * @throws HashException               when the hash methods are not available
+     * @throws BlockAlreadyExistsException when a block already exists
+     * @throws NoSuchAlgorithmException
+     * @throws InvalidKeyException
+     * @throws SignatureException
      */
     public static Block addContact(User user)
-            throws HashException, BlockAlreadyExistsException, NoSuchAlgorithmException, InvalidKeyException, SignatureException {
-
+            throws HashException, BlockAlreadyExistsException, NoSuchAlgorithmException,
+            InvalidKeyException, SignatureException {
         try {
             final byte[] message = user.getPublicKey().getEncoded();
             final byte[] signature = ED25519.generateSignature(message, owner.getPrivateKey());
-            //Adding the user into your own chain
             return blockController.addBlockToChain(user, signature, message);
         } catch (DatabaseException e) {
-            return null;
+            e.printStackTrace();
```

Comment:
Remove this and the next line, just add the DatabaseException to the throws from the method